### ISSUE TITLE
UI views not updated after tile deletion and reuplaod

### ISSUE THREAD

**Issue by rembart** [higlass/higlass-server#132]
Hello,
after deleting a tilesets and re-uploading it with the same UID & filename, the heatmap view of this tilesets has still the old file version (in our case the chromosomes got sorted differently) 
Uploading the file with a new UID/Filename works fine
I assume it's some redis cache, but haven't had the time to have a closer look.

Steps to reproduce:
`curl -u admin:XXXXX -F "datafile=@ABC.mcool" -F "filetype=cooler" -F "datatype=matrix" -F "uid=ABC" -F "coordSystem=mm9" https://higlass.xyz.ac.at/api/v1/tilesets/
curl -u admin:XXXXX -X "DELETE" https://higlass.xyz.ac.at/api/v1/tilesets/ABC/
Change content of ABC.mcool
curl -u admin:XXXXX -F "datafile=@ABC.mcool" -F "filetype=cooler" -F "datatype=matrix" -F "uid=ABC" -F "coordSystem=mm9" https://higlass.xyz.ac.at/api/v1/tilesets/
Check WebUI `

Here is the server setup:
`  - name: 'redis'
    image: 'registry.hub.docker.com/library/redis'
    tag: '5.0.8-alpine'
    uid: 999
    gid: 1000
    ip: '10.88.0.30'
    selinux: true
    volumes:
      - name: 'data'
        path: '/data'
        mode: '0700'
  - name: 'higlass'
    image: 'docker.io/higlass/higlass-docker'
    tag: 'v0.6.43'
    ip: '10.88.0.10'
    uid: 33
    gid: 33
    selinux: true
    volumes:
      - name: 'data'
        path: '/data'
      - name: 'tmp'
        path: '/tmp'
      - name: 'scripts'
        path: '/scripts'
      - name: 'uploads'
        path: '/var/lib/nginx/body'
        mode: '0777'
    environment_variables:
      SITE_URL: "{{ app_generic_fqdn }}"
      UPLOAD_ENABLED: true
      PUBLIC_UPLOAD_ENABLED: false
      REDIS_HOST: '10.88.0.30'
      REDIS_PORT: 6379`

**Comment by pkerpedjiev** [comment]
Yeah, if you have a redis cache running, that's probably the issue. If you want to get rid of all the redis entries you can do something like: `docker exec redis-container redis-cli flushall`

**Comment by rembart** [comment]
Would you accept a pull request to delete the tiles from redis on tileset deletion?
